rootfs/usr/local/bin/aws-config-setup

```diff
@@ -21,6 +21,27 @@ if [ -z "${AWS_ROOT_ACCOUNT_ID}" ]; then
 	exit 1
 fi
 
+if [ ! -d "/localhost/.awsvault/" ] || [ ! -w "/localhost/.awsvault/" ]; then
+	echo "/localhost/.awsvault/ is not writable"
```

Comment:
I don't think this is what we want. If the directory does not exist, then we want `aws-vault` to create it.

Reply:
Yes, but that's `aws-vault` not `aws-config-setup`

Reply:
The `aws-config-setup` script is what calls `aws-vault`. That is, in a cold start scenario a user will never have run `aws-vault add` before so these directories will not exist.

Reply:
Oh, I thought you meant [`aws-vault.sh`](https://github.com/cloudposse/geodesic/blob/258ca72a1f81a086d7e1bfbf3b8c44ee45123fb2/rootfs/etc/profile.d/aws-vault.sh#L56-L57) which will have run and created those directories. `aws-vault` is going to create `~/.awsvault` which is `/conf/.awsvault`, not `/localhost/.awsvault`.

Reply:
Perhaps, but it does not create `.awsvault/keys`. I just tried it on a cold start. It creates `.awsvault` but not `.awsvault/keys`